client/gutenberg/extensions/markdown/markdown-editor.scss

```diff
@@ -0,0 +1,8 @@
+.wp-block-a8c-markdown__placeholder {
+	opacity: 0.5;
+	pointer-events: none;
+}
+.wp-block-a8c-markdown__preview p {
+	min-height: 1.8em;
+	white-space: pre-wrap;
+}
```

Comment:
I took this CSS from the original Jetpack-Markdown PR: https://github.com/Automattic/jetpack/pull/9705